app/routes/topics.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+
+  model: function () {
+    return this.store.findAll('topic');
+  },
+
+  actions: {
+    createTopic (topic) {
+      let controller =  this.get('controller');
+      topic.save().then(function () {
+        controller.replaceNewTopic();
+      }, function () {});
```

Comment:
Is the second function (used to catch errors) needed? I think in JS if you don't use the second param it just goes away.